jenkinsapi_tests/unittests/test_job.py

```diff
@@ -5,19 +5,14 @@
 from . import configs
 from jenkinsapi.job import Job
 from jenkinsapi.build import Build
-from jenkinsapi.jenkins import Jenkins
 from jenkinsapi.jenkinsbase import JenkinsBase
 from jenkinsapi.custom_exceptions import NoBuildData
+from jenkinsapi_tests.conftest import create_jenkins_object
 
 
 @pytest.fixture(scope='function')
 def jenkins(monkeypatch):
-    def fake_poll(cls, tree=None):   # pylint: disable=unused-argument
-        return {}
-
-    monkeypatch.setattr(Jenkins, '_poll', fake_poll)
-    new_jenkins = Jenkins('http://halob:8080/')
-
+    new_jenkins = create_jenkins_object(monkeypatch, 'http://halob:8080/')
```

Comment:
Why this is required? Better approach is to name "create_jenkins_object" as "jenkins", in conftest.py, and remove old fixtures named "jenkins".